core/src/main/java/org/dcache/nfs/v3/NfsServerV3.java

```diff
@@ -21,6 +21,7 @@
 
 import org.dcache.auth.Subjects;
 import org.dcache.nfs.ExportFile;
```

Comment:
is this import still in use?

Reply:
No. Removed that and 2 other unused ones in next commit